marabunta/config.py

```diff
@@ -64,17 +64,36 @@ class EnvDefault(argparse.Action):
 
     def __init__(self, envvar, required=True, default=None, **kwargs):
         if not default and envvar:
-            if envvar in os.environ:
-                default = os.environ[envvar]
+            default = self.get_default(envvar)
         if required and default is not None:
             required = False
         super(EnvDefault, self).__init__(default=default, required=required,
                                          **kwargs)
 
+    def get_default(self, envvar):
+        if envvar not in os.environ:
+            return
+        return os.environ[envvar]
+
     def __call__(self, parser, namespace, values, option_string=None):
         setattr(namespace, self.dest, values)
 
 
+class BoolEnvDefault(EnvDefault):
+
+    def get_default(self, envvar):
+        if envvar not in os.environ:
+            return False
+        val = os.environ[envvar]
+        if isinstance(val, str):
+            if val in ['true', 'True', '1']:
```

Comment:
I would be in favor of using https://docs.python.org/2/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool (on a lowered value)

Reply:
@guewen strtobool raises if he can't match i think we should catch it and set to False, do you agree?